cuda_core/cuda/core/experimental/_stream.pyx

```diff
@@ -103,35 +105,27 @@ class Stream:
 
     """
 
-    class _MembersNeededForFinalize:
-        __slots__ = ("handle", "owner", "builtin")
-
-        def __init__(self, stream_obj, handle, owner, builtin):
-            self.handle = handle
-            self.owner = owner
-            self.builtin = builtin
-            weakref.finalize(stream_obj, self.close)
+    cdef:
+        object _handle
+        object _owner
```

Comment:
Spacing on this line renders differently from spacing of adjacent lines on GH. Not sure why, but should check it out.

Reply:
checks out